src/app/routes/app-routes.tsx

```diff
@@ -41,7 +41,7 @@ import { RequestError } from '@app/pages/request-error/request-error';
 import { BroadcastError } from '@app/pages/send/broadcast-error/broadcast-error';
 import { sendOrdinalRoutes } from '@app/pages/send/ordinal-inscription/ordinal-routes';
 import { sendCryptoAssetFormRoutes } from '@app/pages/send/send-crypto-asset-form/send-crypto-asset-form.routes';
-import { alexSwapRoutes } from '@app/pages/swap/alex-swap-container';
+import { bitflowSwapRoutes } from '@app/pages/swap/bitflow-swap-container';
```

Comment:
Are we replacing `alex-swap-container` with `bitflow-swap-container`? 

Maybe we can get rid of the ALEX code then

Reply:
I would say if it's not much extra work, clear it out and add a comment to https://github.com/leather-io/issues/issues/99 to explain it's being removed. 

We will be able to remove `"alex-sdk": "2.1.3",` then also and close off this https://github.com/leather-io/extension/pull/5852

Reply:
I removed the existing ALEX swap components but the `SwapAsset` interface imported from `mono` has a reference to an ALEX SDK type that needs to be referenced. We should be able to refactor that in `mono` and remove that dependency here but for now it would be difficult to remove the SDK package entirely.

Reply:
Sounds great 👍 Nice work